recipe/setup.cfg

```diff
@@ -0,0 +1,5 @@
+[tool:pytest]
+junit_family = xunit2
```

Comment:
Just for my information. Is this so that azure_pipelines can consume the output?

Reply:
Right. And it seems to be working, which is awesome! Even though the build logs roll off after a few months, the test results persist (for at least a year)... knowing which previous builds were from which PRs might help us avoid future rework.